### PR TITLE
sqlite3: add page

### PR DESCRIPTION
**Pull request by sbrl**
I saw that we didn't have this page yet, and I thought I'd create it :-)

On another (unrelated) note, I'm wondering about whether we could have another themed list of commands like we had in that issue with the like of terminal-based utilities from some article or other (I can't find it, sorry!).

### CODE REVIEW

**Comment by sbrl** [comment]
Umm ping pong @waldyrious @agnivade @anyone-else-able-to-peer-review?

File: pages/common/sqlite3.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+`sqlite3`
+
+- Open an interactive shell against an existing database:
+
+`sqlite3 {{path/to/database.sqlite3}}`
+
+- Execute an SQL statement against a database and then exit:
+
+`sqlite3 {{path/to/database.sqlite3}} '{{SELECT * FROM some_table;}}`
```
Comment (agnivade) [inline pages/common/sqlite3.md]: Isn't the closing quote missing for the sql query ?

Comment (sbrl) [inline pages/common/sqlite3.md]: It is! I'll correct that now. Good spot.

**Comment by waldyrious** [comment]
> Umm ping pong @waldyrious @agnivade @anyone-else-able-to-peer-review?

:sweat_smile: sorry about the silence! FWIW, I'd have nothing to point out for changing here. Really nice contrib.

**Pull request merged**